### PR TITLE
Add comet-ms 2018012

### PR DESCRIPTION
**Pull request by lkuchenb**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by lkuchenb** [comment]
@bioconda/core It would be great if somebody could cross check this

File: recipes/comet/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+chmod 755 comet.2018012.linux.exe
+cp comet.2018012.linux.exe ${PREFIX}/bin/comet.exe
```
[the remark this answers is not in the document]
Comment (Leon-Bichmann) [reply recipes/comet/build.sh]: might be a problem for openms, as here the executable is called comet.exe

https://github.com/OpenMS/OpenMS/blob/9087c9ccac1dcfe70fa1859fc43055f81ebde769/src/tests/topp/THIRDPARTY/third_party_tests.cmake
`# Comet
OPENMS_FINDBINARY(COMET_BINARY "comet.exe" "Comet")`

[the remark this answers is not in the document]
Comment (bgruening) [reply recipes/comet/build.sh]: Can/should we have then a symlink?

[the remark this answers is not in the document]
Comment (lkuchenb) [reply recipes/comet/build.sh]: @bgruening This also bugs me, but even upstream calls the Linux executable '.exe' - it seems to be their preferred naming. A symlink might be a good idea. I'll push again.

**Comment by lkuchenb** [comment]
@bgruening Unless there are any further objections, I'd merge this.

**Comment by bgruening** [comment]
@lkuchenb please! Thanks a lot!

**Pull request merged**